pinc/forum_interface.inc

```diff
@@ -44,7 +46,8 @@ assert(PHPBB_VERSION == 3, "Only phpBB version 3 is supported");
 */
 function phpbb_table($table)
```

Comment:
we're waiting to rename these methods right?

Reply:
No, as the whole file will be renamed `forum_interface_phpbb3.inc` in a follow-up PR so everything in here will be phpBB3-specific.